video/out/gpu/video.c

```diff
@@ -1381,18 +1384,27 @@ next_bind: ;
     return true;
 }
 
+static struct ra_tex **next_hook_tex(struct gl_video *p)
+{
+    MP_TARRAY_GROW(p, p->hook_textures, p->idx_hook_textures);
+    p->num_hook_textures = MPMAX(p->num_hook_textures, p->idx_hook_textures);
```

Comment:
I don't get this.

Reply:
And rightfully so, since it doesn't make any sense. Fixed, should make much more sense now.